README.md

```diff
@@ -147,6 +151,14 @@ The inline version also works:
 <section class="bg-[url('image.svg')]">Has the image as it's background</section>
 ```
 
+### Contributing
```

Comment:
There is a CONTRIBUTING.md file which I think covers this topic. If not, please make changes to that file and not the user-facing README.

Reply:
I've added content addressing how to develop against this gem in e22c8cee